composeApp/src/commonMain/kotlin/com/majotyler/hiittimer/presentation/TimerViewEvent.kt

```diff
@@ -0,0 +1,12 @@
+package com.majotyler.hiittimer.presentation
+
+import androidx.compose.runtime.Immutable
+
+@Immutable
+sealed interface TimerViewEvent {
```

Comment:
This `sealed interface` and especially just the word `sealed` is really important. Remember the inheritance? It's just another, more common way of doing that.

For example this could be `sealed interface Shape { data class Circle(val radius: Int) : Shape }`

We'll do this a ton in the future, so it will make more sense as you see it more.